apps/web/cypress.config.ts

```diff
@@ -12,7 +12,6 @@ export default defineConfig({
   viewportWidth: 1280,
   viewportHeight: 1500,
   video: false,
-  videoUploadOnPasses: false,
```

Comment:
update cypress to v13, `videoUploadOnPasses` is removed and default to false.

![CleanShot 2024-08-21 at 16 50 26](https://github.com/user-attachments/assets/2fc6756e-0adf-4754-b4b2-9369e2093530)